DXRVanilla/DeusEx/Classes/Player.uc

```diff
@@ -2203,6 +2203,16 @@ exec function ActivateBelt(int objectNum)
     }
 }
 
+function GoalCompletedSilent(name goalName)
```

Comment:
this file is only available in vanilla, so probably better to move this function